pixelish/usr/share/asteroid-launcher/watchfaces/pixelish.qml~

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
This .qml~ file is an auto-save development artifact and can be removed, correct?

Reply:
Dammit man... I was triple checking before I committed to make sure that didn't get through, and it still did. Has gedit betrayed me, or have I betrayed myself?

...I betrayed myself, because I was also triple checking the date alignment to commit the build value, and because I was last second changing the preview, I committed the qmlscene value 😔 I'll fix that as well